bc/subscription/models.py

```diff
@@ -187,6 +187,16 @@ def document_link(self) -> Optional[str]:
                 f"{self.subscription.cl_slug}/"
             )
 
+    @property
+    def cl_pdf_or_pacer_url(self) -> str:
+        return f"{self.cl_document_url()}?redirect_to_download=True"
+
+    @property
+    def cl_docket_url(self) -> str | None:
+        if not self.subscription:
+            return None
+        return self.subscription.cl_url()
```

Comment:
Shouldn't this be a property for consistency?

Reply:
You're right!